addons/apollo/functions/fnc_vehicleLoad.sqf

```diff
@@ -47,7 +47,7 @@ if (_retrieveVehicles == "ready") then {
 
     // Fix possible issue where physics don't activate until doing it manually (eg. shooting the object)
     _vehList params ["_vehFixPhysics"];
```

Comment:
Why is that done only for the first vehicle. 🤔 Need to investigate if left-over from initial testing and should indeed happen on all vehicles.